doc/next/_sources/campaign-linkedads.rst.txt

```diff
@@ -0,0 +1,31 @@
+.. _campaign_linkedads_overview:
+
+Campaign Linked Ads
+===================
+
+.. _campaign-linked-ads:
+
+
+Campaign's linkedAds component defines which ads are part of the campaign. It is possible to add / remove (link / unlink) ads from a campaign using
+:ref:`put_campaign_linkedads` / :ref:`delete_campaign_linkedads`. Deleting a linkedAd from a campaign does not remove the ad from your inventory, it merely takes it offline in the context of the particular campaign.
+
+LinkedAds are stored as part of the campaign, and have a reference to an Ad using its ID. A linkedAd consists of two parts:
+the Ad ID and optional overrides from the campaign defaults.
+Using linkedAds, it is possible to add a single ad to multiple campaigns (which may have different budgets and targeting criteria).
```

Comment:
should we also mention ad can be in multiple campaigns as  `(future plans)` like mentioned in campaign-overview.rst.txt  L72